external/fv3fit/fv3fit/pytorch/graph/graph_config.py

```diff
@@ -0,0 +1,60 @@
+import dataclasses
+from typing import Callable
+import torch.nn.functional as F
+import torch
+import torch.nn as nn
+from dgl.nn.pytorch import SAGEConv
+
+
+@dataclasses.dataclass
+class GraphNetworkConfig:
+    """
+    Attributes:
+        n_hidden: maximum number of hidden channels
+        num_blocks: number of nested network (e.g, 2 means N(N(theta)))
+        aggregator: type of aggregator, one of "mean", "gcn", "pool", or "lstm"
+        activation: activation function
+    """
+
+    n_hidden: int = 256
+    num_blocks: int = 1
+    aggregator: str = "mean"
+    activation: Callable = F.relu
```

Comment:
We can ignore it this PR, but this is not actually configurable from yaml. To do so this configuration will need to be a string, and we'll need to do a `getattr(F, activation)` to retrieve the function itself.